PYIC-7334: Add F2F p1 journeys api-tests

AmritSidhu opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->
<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->

## Proposed changes

### What changed

Add F2F p1 journeys api-tests

### Issue tracking
<!-- List any related Jira tickets or GitHub issues -->
<!-- List any related ADRs or RFCs -->
<!-- Delete/copy as appropriate -->

- [PYIC-7334](https://govukverify.atlassian.net/browse/PYIC-7334)

## Checklists

### Environment variables or secrets

<!-- Delete if changes DO include new environment variables or secrets -->
- [x] No environment variables or secrets were added or changed


[PYIC-7334]: https://govukverify.atlassian.net/browse/PYIC-7334?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

AmritSidhu committed:
PYIC-7334: Add F2F p1 journeys api-tests

DanCorderIPV commented on api-tests/features/p1-journeys.feature:
```diff
@@ -192,3 +192,58 @@ Feature: P1 journey
+      | evidence_requested | {"scoringPolicy":"gpg45","strengthScore":2} |
+    Then I get a 'page-face-to-face-handoff' page response
+
+  Scenario: P1 F2F Low confidence journey should have relevent photoid option text
```
Is this test needed? We're not testing any text and the test steps are just a subset of `P1 F2F Journey` above

DanCorderIPV commented on api-tests/features/p1-journeys.feature:
```diff
@@ -192,3 +192,58 @@ Feature: P1 journey
       | evidence_requested | {"scoringPolicy":"gpg45","strengthScore":2} |
     Then I get a 'page-face-to-face-handoff' page response
+
+  Scenario: P1 F2F Support low strength F2F documents for thin file users
```
I realise that you've probably just copied this test name over from an end to end test, but there's no KBV step here so I'm not sure what `thin file` has to do with this test.

blakeyp replied:
I think this is referring to thin _fraud_ file - when a user has a low fraud/activity score and we need to pass a required strength to F2F CRI

blakeyp replied:
will be clearer if we just rename it "thin fraud file" users?

blakeyp commented on api-tests/features/p1-journeys.feature:
```diff
@@ -192,3 +192,58 @@ Feature: P1 journey
+    When I submit an 'end' event
+    Then I get a 'page-ipv-identity-postoffice-start' page response with context 'lastChoice'
+
+  Scenario: P1 F2F Support low strength F2F documents for P2 user
```
This is the regression scenario for P2 but we might already have a pretty identical test to this in the P2 tests? if so I don't think we need this

AmritSidhu committed:
PYIC-7334: Update F2F P1 journey api-tests

Remove 2 redundant test already covered
Change test name to specify fraud thin file
